lib/lintSource.js

```diff
@@ -88,7 +87,7 @@ function lintPostcssResult(
   postcssResult.stylelint.quiet = config.quiet
 
   const newlineMatch = postcssResult.root.toResult().css.match(/\r?\n/)
-  const newline = newlineMatch ? newlineMatch[0] : os.EOL
+  const newline = newlineMatch ? newlineMatch[0] : "\n"
```

Comment:
Ah, I see. Thanks!

Let's decide first what is the most desirable behavior for users. I assumed that falling back to the OS default was the most typical pattern. Do you agree?

If so, then we should find a way to fix the tests without altering the behavior for actual users. Can we mock `os.EOL` in Jest tests to always be `\n`?